odoo/addons/base/tests/test_views.py

```diff
@@ -674,6 +674,86 @@ def test_branding_inherit(self):
             second.get('data-oe-id'),
             "second should come from the extension view")
 
+    def test_branding_inherit_wrapping(self):
```

Comment:
Might be worth adding a test reproducting the original crash if something was crashing?

Reply:
Not sure what you mean, is not this test enough?

Reply:
I don't know, hence my question. Here you're showing that the effect of the change are what you wanted them to be, but not necessarily that the high-level error is fixed.

Incidentally in the commit message you're missing the `#` to reference the original issue, so it won't actually close anything. The last line should be "closes #47642".

Reply:
Well I know that... We just avoid flooding the original issue with work-in-progress branches but now that you did it, I'll change that now :shrug: 

I'll make a high level test too.

Reply:
t'was more of a question than a demand, if you don't think it's necessary don't do it.

Reply:
Something seems wrong trying to make the high level test... but I can't see what at the moment :/ I'll continue working on this another day.